src/renderer/modals/FullNode/helpers.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import { ipcRenderer } from "electron";
+import resolveUserDataDirectory from "~/helpers/resolveUserDataDirectory";
+import { setEnvOnAllThreads } from "~/helpers/env";
+import {
+  editSatsStackConfig,
+  stringifySatsStackConfig,
+  parseSatsStackConfig,
+} from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
+import type { SatsStackConfig } from "@ledgerhq/live-common/lib/families/bitcoin/satstack";
+import path from "path";
+const lssFileName = "lss.json";
+
+export const saveLSS = async (lssConfig: SatsStackConfig) => {
+  const userDataDirectory = resolveUserDataDirectory();
+  const filePath = path.resolve(userDataDirectory, lssFileName);
```

Comment:
I think filePath should be on main thread side. There will never be a reason we want the config file to be somewhere else so I would move out this logic (lssFileName, userDataDirectory, filePath) to be in where the invoked functions are implemented.

Reply:
Makes sense